qemu/tests/cfg/unsafe_rebase_snapshot.cfg

```diff
@@ -0,0 +1,20 @@
+- unsafe_rebase_snapshot:
```

Comment:
'unsafe_rebase_to_no_bakcing_file' would be better?

Reply:
`unsafe_rebase_to_none_exist_bakcing_file`?

Reply:
It's good to me.